default-config.mk

```diff
@@ -0,0 +1,24 @@
+# Do not modify this file, if you want to configure your own environment copy
+# this file in config.mk and modify that file, or defining environment
+# variables using the same names found here.
+
+# Enable services to be run in docker.
+#
+# Possible options: mysql, postgres, minio, inbucket, openldap, dejavu,
+# keycloak and elasticsearch
+#
+# Must be space spearated names.
+#
+# Example: mysql postgres elasticsearch
+ENABLED_DOCKER_SERVICES ?= mysql
+
+# Disable entirely the use of docker
+MM_NO_DOCKER ?= false
+
+# Run the server in the background
+RUN_SERVER_IN_BACKGROUND ?= true
```

Comment:
Could we plan for a more general variable name that might apply to a similar `webapp` change? Maybe `MM_DEVELOP_FOREGROUND`?

Reply:
Mmm... interesting. we can assume that you may want to run things in background or in foreground, and if it is in foreground for example, I can fail if you try to execute `make run` saying that when you are using foreground mode you have to execute both comands independently. What do you think @jwilander? that would fit your workflow, or is not what you expect?